src/workerd/io/trace-stream.c++

```diff
@@ -1209,12 +1213,6 @@ kj::Maybe<kj::Own<tracing::TailStreamWriter>> initializeTailStreamWriter(
     // dropping this lambda.
 
     return !state.closing;
-  }, []() -> kj::Date {
-    // TODO(streaming-tail): Return proper timestamps. This callback is used to
```

Comment:
While developing this, I couldn't find a good way to implement the time source here – the lifetime of the WorkerTracer may not match the IoContext lifetime, so taking its time source doesn't seem workable. Clearly, having this would be the most elegant approach but I don't know where we'd take that time source from.